modules/c++/math/wscript

```diff
@@ -4,7 +4,11 @@ VERSION         = '0.1'
 USELIB          = 'MATH'
 MODULE_DEPS     = 'sys'
 
-options = configure = distclean = lambda p: None
+options = distclean = lambda p: None
+
+def configure(conf):
+    conf.check_cc(function_name='isnan', header_name='math.h',
```

Comment:
Do this like we do in the sys wscript where it makes a math_config.h file.  The reasoning is that that way someone building their application without waf  (j.e. just linking in our stuff) will have the defines they need.